app/views/shared/_basic_prisoner_info.html.erb

```diff
@@ -14,8 +14,11 @@
       <td class="govuk-table__cell table_cell__left_align govuk-!-width-one-half"><%= format_date(@prisoner.date_of_birth) %></td>
     </tr>
     <tr class="govuk-table__row">
-      <td class="govuk-table__cell hmpps-table-cell__key">Prisoner number</td>
-      <td class="govuk-table__cell table_cell__left_align hmpps-table-cell__key"><%= @prisoner.offender_no %></td>
+      <td class="govuk-table__cell">Prisoner number</td>
+      <td class="govuk-table__cell table_cell__left_align">
+        <%= @prisoner.offender_no %>
+        <a class="govuk-link pull-right" href="<%= Rails.configuration.new_nomis_host %>/offenders/<%= @prisoner.offender_no%>/quick-look" target="_blank">View NOMIS profile</a>
```

Comment:
Not sure if it is possible, but it would be pretty awesome if we could have a named route to an external resource... so for instance `new_nomis_profile_url(offender_no)` but I suspect it might mean a controller in an action somewhere.  @TobyRet do you know if this is possible?